### ISSUE TITLE
Relationships: err_file_not_found

### ISSUE THREAD

**Issue by cyborgetron** [schemaspy/schemaspy#569]
## Expected Behavior

I should be able to see the relationships graphically.

## Current Behavior

When I run SchemSpy, there are dozens of errors generating the dot files. Then, when I open the html and click on the "relationships" tab, I get an error page that says "Your file was not found." I have tried using `schemaspy.vizjs` in my config file instead of the path to graphviz, and I have tried using `-renderer :quartz` in the command line and `schemaspy.renderer :quartz` in the config file. Nothing seems to work. I did read something about there being a bug with OSX and the newest version of graphviz, but people were saying they fixed that using `-renderer :quartz`, which has not worked for me. 

My config file is as follows:
```
schemaspy.gv=/usr/local/Cellar/graphviz/2.40.1
schemaspy.t=pgsql
schemaspy.dp=postgresql-42.2.5.jar

schemaspy.host=hostAddress
schemaspy.port=5432
schemaspy.db=dbName
schemaspy.u=userName
schemaspy.p=dbPassword
schemaspy.ssl=true

schemaspy.o=dbschema

schemaspy.s=schemaName
```

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Context

* Database type (e.g. MySQL): PostgreSQL 10.7
* Database driver version (is you use a special one): postgresql-42.2.5
* Your DDL (sample how to reproduce the issue):
* Full command including arguments how you run SchemaSpy: pathToSchemaSpy$ java -jar schemaspy-6.0.0.jar -configFile config.file

## Your Environment

* Version used: SchemaSpy 6.0.0
* Environment name and version (e.g. Chrome 39): Chrome 74
* Operating System and version (e.g. Windows 10): Mac OS 10.13.3
* Java version: 1.8.0_161
* Graphviz: 2.40.1 (installed with Brew)

**Comment by npetzall** [comment]
If you can post one of the errors that might help.

`-vizjs` is only available in 6.1.0-SNAPSHOT

Yeah osx has seen a lot of hardship when it comes to graphviz. 

So I'd recommend either using the docker image or use 6.1.0-SNAPSHOT and the `-vizjs` option.

**Comment by npetzall** [comment]
High Sierra `-render :cairo` might work. But if graphviz is installed with brew they changed the formula without any instructions on how to gain the same result as was available.

But since I don't know what part of the system is missing the file I can't help you. I need some log data. 

For more details you can add `-debug`

**Comment by cyborgetron** [comment]
> High Sierra `-render :cairo` might work. But if graphviz is installed with brew they changed the formula without any instructions on how to gain the same result as was available.
> 
> But since I don't know what part of the system is missing the file I can't help you. I need some log data.
> 
> For more details you can add `-debug`

I will give the 6.1.0-SNAPSHOT a try. I did notice that it says `unable to validate png renderer ':cairo'. Reverting to default renderer for png.` I have also run it a few times in debug mode, and I can do it again if the other fixes you suggested don't work.

Here is the log data (I removed a bunch of names and replaced them with jibberish. The databse is very private, so I just can't take any chances):
```
INFO  - Starting Main v6.0.0 on MacBook-Pro.local with PID xxxx (/Users/xxx/schemaspy-6.0.0.jar started by xxx in /xxx/SchemaSpy)
INFO  - The following profiles are active: default
INFO  - Found configuration file: config.file
INFO  - Started Main in 2.8 seconds (JVM running for 3.898)
INFO  - Starting schema analysis
INFO  - Failed to validate png renderer ':cairo'.  Reverting to default renderer for png.
INFO  - Connected to PostgreSQL - 10.7 (Ubuntu 10.7-1.pgdg16.04+1)
INFO  - Gathering schema details
Gathering schema details...WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."api_client" where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.' using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”class" where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."guidebook" where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.honest_conversation_meeting' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.meeting' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx.xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."lead" where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.client' using built-in query with 'count(*)'
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."benchmark_asset_allocation" where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.goals_assumptions_request' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyy” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."pal_item" where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyy” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.xxx’ using built-in query with 'count(*)'
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”xxx” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx' using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyyy” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyy” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyy” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyy” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyyy” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.myyy’ using built-in query with 'count(*)'
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."pal_template" where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."positions" where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyyy” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."ppp” where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”lll" where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyy” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yym' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.othns' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.othytions' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.tr_card' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.preycard' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.prtyplate' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.wretylates' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.phgde' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.plates' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.ples' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.pates' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.pgdfshplates' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.pqwe’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.otaccounts' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.pwqgal' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.othunts' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.portjhgfdsl' using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”qwegions" where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.portfolio_fees' using built-in query with 'count(*)'
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."py" where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.es' using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.plder' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.poqder' using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."secrets" where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."strategies" where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."yyy” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.yyyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx”.”yyy” where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.yyy’ using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."user" where 0 = 1
WARN  - Failed to fetch number of rows for 'xxx.strategies' using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for 'xxx.strategies' using built-in query with 'count(1)'
.WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from "xxx"."versioned_static_content" where 0 = 1
WARN  - Failed to determine auto increment status: {}
WARN  - SQL: select * from “xxx”.”task" where 0 = 1
WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(1)'
.WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(*)'
WARN  - Failed to fetch number of rows for ‘xxx’ using built-in query with 'count(1)'
.(75sec)
Connecting relationships.....................................................(17sec)
Writing/graphing summary.INFO  - Gathered schema details in 17 seconds
INFO  - Writing/graphing summary
....org.schemaspy.util.Dot$DotFailure: 'dot -Tpng:gd dbschema/diagrams/summary/relationships.real.compact.dot -odbschema/diagrams/summary/relationships.real.compact.png -Tcmapx' failed with return code 139
.org.schemaspy.util.Dot$DotFailure: 'dot -Tpng:gd dbschema/diagrams/orphans/api_client.1degree.dot -odbschema/diagrams/orphans/api_client.1degree.png -Tcmapx' failed with return code 139
.....(4sec)
Writing/diagramming detailsINFO  - Completed summary in 4 seconds
INFO  - Writing/diagramming details
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
..ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
..ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
.ERROR - There was an error writing a dot file
(7sec)
Wrote relationship details of 50 tables/views to directory 'dbschema' in 107 seconds.
View the results by opening dbschema/index.html
INFO  - Wrote table details in 7 seconds
INFO  - Wrote relationship details of 50 tables/views to directory 'dbschema' in 107 seconds.
INFO  - View the results by opening dbschema/index.html
```

**Comment by npetzall** [comment]
https://github.com/schemaspy/schemaspy/issues/33#issuecomment-382184166

I think that issue was related to return code 139. And comment after the link are a good read. 

People have had luck with macports, and some with brew.

Graphviz is it from brew, macports or did you build it yourself?

**Comment by cyborgetron** [comment]
> [#33 (comment)](https://github.com/schemaspy/schemaspy/issues/33#issuecomment-382184166)
> 
> I think that issue was related to return code 139. And comment after the link are a good read.
> 
> People have had luck with macports, and some with brew.
> 
> Graphviz is it from brew, macports or did you build it yourself?

Thanks a bunch. I will check those out. 

Graphviz is from brew

**Comment by cyborgetron** [comment]
> [#33 (comment)](https://github.com/schemaspy/schemaspy/issues/33#issuecomment-382184166)
> 
> I think that issue was related to return code 139. And comment after the link are a good read.
> 
> People have had luck with macports, and some with brew.
> 
> Graphviz is it from brew, macports or did you build it yourself?

Quick question - where do I find 6.1.0-SNAPSHOT? Is it in the git repo?

**Comment by npetzall** [comment]
There's a link in the overview/readme.

https://github.com/schemaspy/schemaspy/blob/master/README.md#latest-build

**Comment by cyborgetron** [comment]
> There's a link in the overview/readme.
> 
> https://github.com/schemaspy/schemaspy/blob/master/README.md#latest-build

ah there it is! Thanks a ton

**Comment by cyborgetron** [comment]
> There's a link in the overview/readme.
> 
> https://github.com/schemaspy/schemaspy/blob/master/README.md#latest-build

Just to let you know, I got everything working! So the solution was using 6.1.0-SNAPSHOT and vizjs instead of graphviz. I need to spend a little more time checking through everything in the output to make sure it's satisfactory, but I can tell you that everything that should be working seems to be working. 

Thanks again, hopefully this thread can help some other OSX users who are having trouble. SchemaSpy is great. Our developers, a bunch of the devs at our client's company and our data team were impressed when I showed them what it could do.
